AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -335,6 +335,8 @@ hg6384.com-www.6u7u.com##.focus-wrap
 hg6384.com-www.6u7u.com##.copyright-tips
 fdoctor.ru##section.app
 kobi8.com##div[id^="dow_APP"]
+m.tokopedia.com##div[data-testid="divOverlay"]
```

Comment:
Please test your rules in user rules before make pull request. This rule hides overlay only, popup is still visible and scrolling is blocked. And this popup can appear with another classes. Please add these rules:

```
m.tokopedia.com#$#div[data-testid="divOverlay"] { display: none!important; }
m.tokopedia.com#$#div[data-testid="divBottomSheetWebToApp"] { display: none!important; }
m.tokopedia.com#$#body { overflow: visible!important; }
m.tokopedia.com#$#.unf-overlay { display: none!important; }
m.tokopedia.com#$?#body > div > div.unf-sheet__wrapper:has(> article img[alt="app"]) { display: none!important; }
```
If we need to use css rules to unblock scrolling we must use css for all rules to avoid incorrect blocking on platforms which don't support css rules. (otherwise f.e. on iOS popup will be hidden but scrolling will be broken)

Reply:
Okay.

Reply:
8249dbc55a2da08cbefa320d5304d99e2142ea84

Reply:
@piquark6046 thank you)